source/OutfitterPanel.cpp

```diff
@@ -980,3 +980,26 @@ const vector<Ship *> OutfitterPanel::GetShipsToOutfit(bool isBuy) const
 
 	return shipsToOutfit;
 }
+
+
+
+int OutfitterPanel::FindItem(const string &text) const
+{
+	int bestIndex = 9999;
+	int bestItem = -1;
+	vector<Zone>::const_iterator it = zones.begin();
+	for(unsigned int i = 0; i < zones.size(); ++i)
```

Comment:
You can chain multiple iterators together in a for statement.
```suggestion
	for(unsigned int i = 0; i < zones.size(); ++i, ++it)
```

Reply:
Oh awesome, I didn't realise this was possible